DataDog/datadog-agent
# Implements file reading for peer tags to synchronize cross repo usage

### What does this PR do?

Make the `defaultPeerTags` list read from a file so that file can be synchronized with other repos that use peer tags

Part of a larger set of changes to come where common tag names are sourced from a single source of truth across DD repos 

The planned source of truth will generate static files which can be read by the agent code and synchronized across all DD repos involving peer tags

### How to test

Ensure that service map [behavior](https://docs.datadoghq.com/tracing/services/services_map/) is unchanged namely the features related to `peer.*` tags. Specifically, the connections between services in the service map remain unchanged


Run `TestPreparePeerTags` in `concentrator_test.go`
